### DISCUSSION TITLE
Hi thank you for a great software These [Sep 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6408] [Discord Threads]
> 🙋‍♂️ **vadminas**  
#### 1. Swagger started showing random ID instead of Project Name with last updates, i.e. existing endpoints have changed. Is this intentional and will it remain so in the future?  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1149657758971461642/image.png)

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### We had replaced `title` with `id` to keep it same even after changing the project title  
#  
> 👨‍💻 **pranavxc**  
#### Actually we support both title and id in api path but it recommended to use id over title since in future we are planning to remove title support  
#  
> 🙋‍♂️ **vadminas**  
#### Got it, thank you!  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1149657759122473031)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
